dev-python/traits/traits-6.1.1.ebuild

```diff
@@ -15,28 +15,20 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
-IUSE="test"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
```

Comment:
This package is maintained by @TheChymera and has been updated quite recently. 
I am not sure I want to update this without his input, also given that this is dropping tests. (?)

Reply:
> Still, this one can be done by @TheChymera

Sure, that's alright.

At the moment I'm just going through the repoman output from top to bottom. This one only had a minor warning if I remember correctly, but I fixed some other things while I was at it.